{"name": "test_row_count[19615]", "status": "failed", "statusDetails": {"message": "AssertionError: assert 19614 == 19615", "trace": "cursor = <pyodbc.Cursor object at 0x000001EF0715C9B0>, expected_count = 19615\n\n    @pytest.mark.person\n    @pytest.mark.parametrize(\"expected_count\", [19614, 19615, 0])\n    def test_row_count(cursor, expected_count):\n        \"\"\"Check row count in Person.Address table\"\"\"\n        cursor.execute('Select Count(*) from [Person].[Address]')\n        records = cursor.fetchall()\n        r = records[0][0]\n>       assert r == expected_count\nE       assert 19614 == 19615\n\ntests\\test_person_address.py:12: AssertionError"}, "description": "Check row count in Person.Address table", "parameters": [{"name": "expected_count", "value": "19615"}], "start": 1697261645034, "stop": 1697261645035, "uuid": "d007fa06-75b4-4cd0-9887-4da69e797987", "historyId": "f37e08c08e72f0c7776f2906d9f3edea", "testCaseId": "260a372c291b3e25faff80110b896e49", "fullName": "tests.test_person_address#test_row_count", "labels": [{"name": "tag", "value": "person"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_person_address"}, {"name": "host", "value": "DESKTOP-HK54S3B"}, {"name": "thread", "value": "8596-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_person_address"}]}